addons/web/static/src/core/py_js/py_parser.js

```diff
@@ -27,8 +27,10 @@ import { binaryOperators, comparators } from "./py_tokenizer";
  * @typedef {{type: 13, condition: AST, ifTrue: AST, ifFalse: AST}} ASTIf
  * @typedef {{type: 14, op: string, left: AST, right: AST}} ASTBooleanOperator
```

Comment:
seems ok to me. probably not incredibly performant, but not sure that it matters.  My main concern is that it is a lot of complexity, and I would like not to have to support it, but if odoo decides that that's we want to do, then so be it

Reply:
I have removed the changes in py.js and I kept them there https://github.com/odoo/odoo/pull/139223. I propose to merge the Expression editor without those changes (it makes sense without them).